ts/components/wallet/card/index.tsx

```diff
@@ -263,7 +269,8 @@ const mapStateToProps = (
 };
 
 const mapDispatchToProps = (dispatch: Dispatch): ReduxMappedDispatchProps => ({
-  setFavoriteCard: (item: Option<number>) => dispatch(setFavoriteWallet(item))
+  setFavoriteCard: (item: Option<number>) => dispatch(setFavoriteWallet(item)),
```

Comment:
what's the meaning of calling `setFavoriteCard` with `none` ??? or dispatching `setFavoriteWallet` with `none` ?

Reply:
It makes sense if you want to unset the currently favorite card (and not selecting another one)